Yoast/yoast-components
# Improve the check for non-existing field components

The wizard kept loading when a component did not exist. To prevent this, the check if components exist is improved.

The wizard should now just go to the next step and not render the non-existing component.

Testing:
1. Set a component in the production-config.js to a non existing one.
2. Start the wizard and go to the step with the non existing component.
3. The step should just render.
